### PR TITLE
Update labeler.yml

### PR DESCRIPTION
**Pull request by devoncarew**
- update labeler.yaml (to try to fix the CI)

---

- [x] I’ve reviewed the contributor guide and applied the relevant portions to this PR.

<details>
  <summary>Contribution guidelines:</summary><br>

- See our [contributor guide](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md) for general expectations for PRs.
- Larger or significant changes should be discussed in an issue before creating a PR.
- Contributions to our repos should follow the [Dart style guide](https://dart.dev/guides/language/effective-dart) and use `dart format`.
- Most changes should add an entry to the changelog and may need to [rev the pubspec package version](https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#making-a-change).
- Changes to packages require [corresponding tests](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md#Testing).

Note that many Dart repos have a weekly cadence for reviewing PRs - please allow for some latency before initial review feedback.
</details>

### CODE REVIEW

**Comment by dcharkes** [comment]
If you succeed, we can close this:

* https://github.com/dart-lang/native/pull/81

Possibly relevant:

* https://github.com/flutter/flutter/pull/129676

File: .github/workflows/pull_request_label.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/labeler@4f052778de9a9b80cb16cfb9079b02287285a4cb
+      - uses: actions/labeler@ac9175f8a1f3625fd0d4fb234536d26811351594
```
Comment (devoncarew) [inline .github/workflows/pull_request_label.yml:19]: The blast tool bumped this version to a pre-release; this reverts to the latest stable version of the action.

https://github.com/dart-lang/ecosystem/issues/108

**Review by dcharkes — APPROVED** [review]

**Comment by dcharkes** [comment]
I guess we have to merge before it runs the right version of the action? It is still failing.

**Comment by devoncarew** [comment]
> I guess we have to merge before it runs the right version of the action? It is still failing.

I think that's correct (though I was trying to work through when github uses the workflow versions from main and when the versions from the PR).

**Pull request merged**